src/Table/__tests__/Table-test.js

```diff
@@ -49,4 +49,32 @@ describe('Table', function () {
     expect(this.callback).toHaveBeenCalled();
   });
 
+  describe('emptyMessage prop', function () {
+    beforeEach(function () {
+      this.emptyMessage = 'This is a custom message';
+      this.instance = TestUtils.renderIntoDocument(
+        <Table
+          className="table"
+          columns={MockTable.columns}
+          data={[]}
+          emptyMessage={this.emptyMessage}
+          idAttribute={this.idAttribute}
+          sortBy={this.sortBy}
+          onSortCallback={this.callback} />
+      );
+    });
+
+    it('should display the custom empty message', function () {
```

Comment:
Can we also have a test case for the inverse? To make sure that `'No data'` is used?